db/migrate_cards/20160107165153_import_general_overview_related.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportGeneralOverviewRelated < Card::Migration
+  def up
+    import_json "general_overview_related.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.